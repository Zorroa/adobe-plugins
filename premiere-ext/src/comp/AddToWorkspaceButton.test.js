import renderer from "react-test-renderer"
import AddToWorkspaceButton from "./AddToWorkspaceButton"

describe("<AddToWorkspaceButton/>", () => {
  it("should render properly", () => {
    const wrapper = renderer.create(
        <AddToWorkspaceButton onpress={()=>{}} />
    )
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
