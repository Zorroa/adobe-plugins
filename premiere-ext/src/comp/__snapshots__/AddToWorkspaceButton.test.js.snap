// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddToWorkspaceButton/> should render properly 1`] = `
<button
  className="btn btn-dark btn-md btn btn-primary"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  <svg
    className="bi bi-download"
    fill="currentColor"
    height="1em"
    viewBox="0 0 20 20"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M.5 9.9a.5.5 0 0 1 .5.5v2.5a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1v-2.5a.5.5 0 0 1 1 0v2.5a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2v-2.5a.5.5 0 0 1 .5-.5z"
      fillRule="evenodd"
    />
    <path
      d="M7.646 11.854a.5.5 0 0 0 .708 0l3-3a.5.5 0 0 0-.708-.708L8.5 10.293V1.5a.5.5 0 0 0-1 0v8.793L5.354 8.146a.5.5 0 1 0-.708.708l3 3z"
      fillRule="evenodd"
    />
  </svg>
    Add to workspace
</button>
`;
